Terminal.Gui/Core/Border.cs

```diff
@@ -899,19 +939,48 @@ private void AddRuneAt (ConsoleDriver driver, int col, int row, Rune ch)
 			driver.AddRune (ch);
 		}
 
+		/// <summary>
+		/// Drawn the view title from a <see cref="View"/>.
```

Comment:
Should be

`Draws the <see cref="View.Title"> to the screen.`

Reply:
Done. Thanks.